src/sentry/tasks/delete_seer_grouping_records.py

```diff
@@ -95,30 +85,11 @@ def may_schedule_task_to_delete_hashes_from_seer(group_ids: Sequence[int]) -> No
     ):
         return
 
-    batch_size = options.get("embeddings-grouping.seer.delete-record-batch-size")
-
-    group_hashes = get_hashes_for_group_ids(group_ids, project)
-    if not group_hashes:
-        return
-
-    # Schedule tasks in chunks
-    for i in range(0, len(group_hashes), batch_size):
-        chunk = group_hashes[i : i + batch_size]
-        delete_seer_grouping_records_by_hash.apply_async(args=[project.id, chunk, 0])
-
-
-def get_hashes_for_group_ids(group_ids: Sequence[int], project: Project) -> list[str]:
-    hashes_batch_size = options.get("deletions.group-hashes-fetch-batch-size")
-    group_hashes = []
-
-    for group_id in group_ids:
-        for group_hash in RangeQuerySetWrapper(
```

Comment:
`RangeQuerySetWrapper` adds a sort on `id` and there's no compound index of `project_id`, `group_id` and `id`.